lib/libutils/ext/include/fault_mitigation.h

```diff
@@ -250,7 +250,7 @@ extern struct ftmn_func_arg *__ftmn_global_func_arg;
 static inline struct ftmn_func_arg **__ftmn_get_tsd_func_arg_pp(void)
 {
 #if defined(CFG_FAULT_MITIGATION) && defined(__KERNEL__)
-    if (thread_get_id_may_fail() >= 0)
+	if (thread_get_id_may_fail() >= 0)
```

Comment:
I'd prefer to see this in a separate commit since it's a generic change, not a riscv change.

Reply:
I submitted another PR https://github.com/OP-TEE/optee_os/pull/6356